src/components/Button/ButtonAddPostTopic.js

```diff
@@ -42,12 +51,12 @@ const styles = StyleSheet.create({
     zIndex: 99,
     backgroundColor: COLORS.black
   },
-  buttonContainer: {
-    backgroundColor: COLORS.signed_primary,
+  buttonContainer: (followType) => ({
+    backgroundColor: followType === 'incognito' ? COLORS.anon_secondary : COLORS.signed_primary,
     // height: dimen.size.TOPIC_FEED_POST_BUTTON_HEIGHT,
     borderRadius: dimen.size.FEED_ACTION_BUTTON_RADIUS,
     justifyContent: 'center'
-  },
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styling of the button container now depends on the `followType`. This is a good use of dynamic styling based on props. Just ensure that the `followType` value always matches the expected values ('incognito' or others).
```diff
- buttonContainer: {
-   backgroundColor: COLORS.signed_primary,
+ buttonContainer: (followType) => ({
+   backgroundColor: followType === 'incognito' ? COLORS.anon_secondary : COLORS.signed_primary,
    borderRadius: dimen.size.FEED_ACTION_BUTTON_RADIUS,
    justifyContent: 'center'
- },
+ }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->